lib/readline.js

```diff
@@ -301,6 +303,11 @@ Interface.prototype.question = function(query, cb) {
       this.prompt();
     }
   }
+
+  return new Promise((resolve, reject) => {
```

Comment:
I think only return Promise when no parameter `cb`.

Reply:
This is probably a good idea, although there could be some edge cases where both are needed. I can only think of one reason to keep it, and that would be for slightly shortening code:

```js
const someFunction = require('somefunction')

async someAsyncFunction() {
    const answer = await question('What do you think of node.js')
    console.log(`thank you for your feedback: ${answer}`)
    someFunction(answer)
}

//becomes

async someAsyncFunction() {
    someFunction(await question('What do you think of node.js', answer => console.log(`thank you for your feedback: ${answer}`)))
}
```

It's probably better the first way, but some people might want to use it the second way.

Reply:
but in a second way, we don't know which will run firstly. This example must be a nightmare

```js
someFunction1(await question('What do you think of node.js', answer => someFunciton2()))
```